src/style.css

```diff
@@ -0,0 +1,29 @@
+html body {
```

Comment:
It is enough to reset body margin and padding
```suggestion
body {
```